lib/_http_client.js

```diff
@@ -471,29 +470,22 @@ function parserOnIncomingClient(res, shouldKeepAlive) {
     // We already have a response object, this means the server
     // sent a double response.
     socket.destroy();
-    return;
+    return 0;  // No special treatment.
```

Comment:
Before anyone asks - yes, these could be constants, please suggest good names. :-)

(They don't have names in http_parser either though.)

Reply:
Just made these up on the spot. Feel free to ignore. How about `kSkipBodyAsUpgrade`, `kSkipBodyNoUpgrade`... and I don't know for the 0 case.